substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -274,9 +275,6 @@ public static void setOptimizeValueUpdateHandler(ValueUpdateHandler<Optimization
     @Option(help = "Path passed to the linker as the -rpath (list of comma-separated directories)")//
     public static final HostedOptionKey<LocatableMultiOptionValue.Strings> LinkerRPath = new HostedOptionKey<>(LocatableMultiOptionValue.Strings.buildWithCommaDelimiter());
 
-    @Option(help = "Directory of the image file to be generated", type = OptionType.User)//
-    public static final HostedOptionKey<String> Path = new HostedOptionKey<>(null);
```

Comment:
`Path` has  a specific meaning in the context of NI and I don't think that moving it to `CommonOptions` is a good idea. I think it's better if you add a specific options (e.g., `ReportsPath`) somewhere in the standalone project and use that instead.

Reply:
Fixed. Move the `ReportsPath` option into `StandaloneOptions` as it is only used by standalone pointsto, it's not necessary to put it in the common place.